rootfs/scheduler/k8s.py

```diff
@@ -470,16 +502,13 @@ def _create_rc(self, name, image, command, **kwargs):
                 break
 
             time.sleep(1)
-
         return resp.json()
 
     def create(self, name, image, command, **kwargs):
         """Create a container."""
         logger.debug('create {}, img {}, params {}, cmd "{}"'.format(name, image, kwargs, command))
-        self._create_rc(name, image, command, **kwargs)
-        image = self.registry + '/' + image
```

Comment:
or at least just move this past `_create_rc()` so `_create_service()` has the right value